client/src/modules/enterprises/enterprises.html

```diff
@@ -61,13 +61,19 @@
                       <i class="fa fa-edit"></i> <span translate>ENTERPRISE.UPDATE_LOGO</span>
                       <input
                         type="file"
+                        name="logo"
                         accept="image/*"
                         ng-model="EnterpriseCtrl.file"
                         ngf-max-size="2MB"
                         ngf-select="EnterpriseCtrl.setThumbnail($file)"
                         style="display: none;">
                     </label>
                   </div>
+                  <div class="help-block text-center form-warning" data-error-message
+                    ng-show="EnterpriseForm.logo.$error.maxSize">
```

Comment:
This is good!  Great catch.

By the way ... you could put this into [our messages.tmpl.html](https://github.com/IMA-WorldHealth/bhima/blob/f35a8dfdf0556bda0bc35da9e805c58641e45ce2/client/src/modules/templates/messages.tmpl.html) file so that all inputs that error on `maxSize` benefit from it.  You'd need to add the following code:

```html
<div class="help-block" ng-messages="EnterpriseForm.logo.$error" ng-show="EnterpriseForm.$submitted">
<div ng-messages-include="modules/templates/messages.tmpl.html"></div>
</div>
```

to this HTML file to make it work, right below the form input.

Reply:
I'll leave it up to you to decide if you want to do that here or if we want to push that off for a future PR.